src/include/network/lan/hardware.rb

```diff
@@ -612,9 +596,7 @@ def handleHW(_key, event)
           UI.ChangeWidget(
             Id(:ifcfg_name),
             :Items,
-            LanItems.FreeDevices(@hardware["realtype"]).map do |index|
-              @hardware["realtype"] + index
-            end
+            LanItems.new_type_devices(@hardware["realtype"], 10)
```

Comment:
and here again that `10` so really I would like to see constant, so it can be change in one run if decided in future.

Reply:
dtto